README.md

```diff
@@ -22,3 +29,41 @@ and this to your crate root:
 ```rust
 extern crate uuid;
 ```
+
+## Examples
+
+To create a new random (V4) UUID and print it out in hexadecimal form:
+
+```rust
+use uuid::Uuid;
+
+fn main() {
+    let my_uuid = Uuid::new_v4();
+    println!("{}", my_uuid.to_string());
```

Comment:
You should be able to elide the `.to_string()` here and just have it printed directly